server/src/main/scala/drt/staff/ImportStaff.scala

```diff
@@ -0,0 +1,31 @@
+package drt.staff
+
+import org.joda.time.{DateTime, DateTimeZone}
+import play.api.libs.json.{JsResult, JsValue, Json, Reads}
+import services.SDate
+import services.SDate.implicits._
+
+import scala.util.parsing.json.JSON
+
+case class StaffShift(port_code: String, terminal: String, staff: String, shift_start: String)
+
+case class StaffShifts(shifts: List[StaffShift])
+
+object ImportStaff {
+  def staffJsonToShifts(staffJson: JsValue): Option[String] = {
+    implicit val shiftFormat = Json.format[StaffShift]
+    implicit val shiftsFormat = Json.format[StaffShifts]
+    staffJson.validate[StaffShifts].asOpt map {
+      case StaffShifts(shifts) =>
+        shifts.zipWithIndex.map{
+          case (shift, index) =>
+            //The client deals in local time, and these shifts are sent to the client as strings with no timezone for now.
+            //TODO: store shifts not as strings.
+            val shiftStartDate = new DateTime(shift.shift_start).withZone(DateTimeZone.forID("Europe/London"))
```

Comment:
it won't - because we control the poster, but what happens if the date string is not formatted correctly?